pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -401,7 +401,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env *model.En
 		}
 		// Construct the default filter chain.
 		if len(allChains) == 0 {
-			log.Infof("Use default filter chain for %v", endpoint)
+			log.Warnf("Use default filter chain for %v", endpoint)
```

Comment:
how much log spam do we expect from this?

Reply:
For now, it depends whether `PERMISSIVE` is turned on for a listener. I thought `Warn` log won't be  printed by pilot's default logging level?

Reply:
My question is more about how often this will be printed. If it's a common case, it will likely fill Pilot's logs.

Reply:
Yes, `warn` will be printed by default.  I think we should just leave it at `info`.